packages/ra-core/src/dataProvider/useLoading.ts

```diff
@@ -17,5 +17,5 @@ import { useIsFetching, useIsMutating } from 'react-query';
 export const useLoading = () => {
     const isFetching = useIsFetching();
     const isMutating = useIsMutating();
-    return isFetching || isMutating;
+    return isFetching > 0 || isMutating > 0;
```

Comment:
Why this change?

Reply:
Because they return numbers where we expect a boolean